src/Tools/Source/RunTests/ProcDumpUtil.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace RunTests
+{
+    internal struct ProcDumpInfo
+    {
+        private const string s_KeyProcDumpFilePath = "ProcDumpFilePath";
+        private const string s_KeyDumpDirectory = "ProcDumpOutputPath";
```

Comment:
The `s_` has always been used for private constants. See the coding style 

https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md